third_party/move/move-prover/tests/sources/functional/ModifiesErrorTest.v2_exp

```diff
@@ -1,64 +1,4 @@
 Move prover returns: exiting with verification errors
-warning: Unused assignment to `x0`. Consider removing or prefixing with an underscore: `_x0`
```

Comment:
Warnings are removed because these variables appear in the spec, which is consistent with the behavior of the prover when using V1 compiler.